jkube-kit/jkube-kit-wildfly-jar/src/test/java/org/eclipse/jkube/wildfly/jar/generator/WildflyJARGeneratorTest.java

```diff
@@ -54,8 +52,8 @@ class WildflyJARGeneratorTest {
 
     @BeforeEach
     void setUp() {
-      context = mock(GeneratorContext.class);
-      project = mock(JavaProject.class);
+        project = JavaProject.builder().build();
```

Comment:
The project variable is no longer used and can be removed (also from line 51)

Reply:
making the changes now.